src/start_server.cr

```diff
@@ -7,6 +7,18 @@ if Lucky::Env.development?
   Avram::SchemaEnforcer.ensure_correct_column_mappings!
 end
 
+Avram::QueryEvent.subscribe do |event|
+  req = Fiber.current.breeze_request
```

Comment:
This gets the request that is set in `store_breeze_request` if possible. That way you can view all sql that was run inside a request.